### PR TITLE
Dev/update smb slam

### PR DESCRIPTION
**Pull request by nubertj**
Removed legacy packages and added everything to  create map with open3d_slam, both when reading from rosbag, as well as when running it as a node.

### CODE REVIEW

**Review by mantelt — COMMENTED** [review]
thanks for cleaning up smb_slam,  @nubertj!

In general this looks good to me, but I added a few comments, mostly FYI. 

Also, I think it would be good practice to add a doc="" attribute to the arg tag, to make clear what the argument is used for.

**Comment by nubertj** [comment]
thanks for your comments, they make a lot of sense. I will update it tonight!
Best,
Julian

File: smb_slam/config/open3d_slam/params_robosense_rs16.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    max_n_iter: 50
+  scan_processing:
+    voxel_size: -1.0
+    downsampling_ratio: 0.45
+    scan_cropping:
+      cropping_radius_max: 40.0
+      cropping_radius_min: 0.0
+      min_z: -30.0
+      max_z: 30.0
+      cropper_type: "MaxRadius" #options are Cylinder, MaxRadius, MinRadius, MinMaxRadius
```
[the remark this answers is not in the document]
Comment (nubertj) [reply smb_slam/config/open3d_slam/params_robosense_rs16.yaml:15]: I did not see any artifacts, but I also did not inspect the map super closely.
I just built it, and then ran ICP_localization over it.

File: smb_slam/launch/build_map.launch [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
+  <arg name="parameter_folder_path" default="$(find smb_slam)/config/open3d_slam/"/>
+  <arg name="parameter_filename"    default="params_robosense_rs16.yaml"/>
+  <arg name="read_from_rosbag"      default="false"/>
+  <arg name="rosbag_filepath"       default=""/> # Can be used to process rosbag sequentially, usually much faster without missing messages
+  <arg name="launch_rviz"           default="false"/>
+  <arg name="launch_sensors"        default="false"/>
+  <arg name="use_sim_time"          default="false"/>
 
+  <!-- Potentially launch sensors -->
+  <include file="$(find smb)/launch/sensors.launch" if="$(arg launch_sensors)"> </include>
```
[the remark this answers is not in the document]
Comment (nubertj) [reply smb_slam/launch/build_map.launch:15]: I did not launch this one, it was just there in the old rosbag (with compslam), so I kept it. 
I guess having the sensors should be good enough, as extrinsics for SE are hardcoded, and open3d_slam only requires lidar. So for building the map it should be all good.

[the remark this answers is not in the document]
Comment (mantelt) [reply smb_slam/launch/build_map.launch:15]: agreed

**Review by mantelt — APPROVED** [review]
Thanks for looking into this! lgtm now.

**Pull request merged**